ibis/expr/api.py

```diff
@@ -324,6 +325,29 @@ def table(schema: sch.Schema, name: str | None = None) -> ir.TableExpr:
     return node.to_expr()
 
 
+def local_table(schema: sch.Schema, name: str | None = None) -> ir.TableExpr:
+    """Create a local table for build expressions without data.
```

Comment:
This is exactly what `ibis.table` does. Can you use that instead?

Reply:
Not related to UDT; please skip.